scripts/resources/util.py

```diff
@@ -271,4 +287,118 @@ def area_median_income():
           '<100': ['Owner 0-30', 'Owner 30-50', 'Owner 50-80', 'Owner 80-100', 'Renter 0-30', 'Renter 30-50', 'Renter 50-80', 'Renter 80-100'],
           '<120': ['Owner 0-30', 'Owner 30-50', 'Owner 50-80', 'Owner 80-100', 'Owner 100-120', 'Renter 0-30', 'Renter 30-50', 'Renter 50-80', 'Renter 80-100', 'Renter 100-120'],
           'all': ['Owner 0-30', 'Owner 30-50', 'Owner 50-80', 'Owner 80-100', 'Owner 100-120', 'Owner 120+', 'Renter 0-30', 'Renter 30-50', 'Renter 50-80', 'Renter 80-100', 'Renter 100-120', 'Renter 120+']
-          }
\ No newline at end of file
+          }
+
+class UtilityBillCalculations():
+
+  @staticmethod
+  def state_level_rates(df):
+    import os
```

Comment:
Add docstrings describing inputs and outputs please, including conversion from average to marginal electricity and natural gas rates using assumed $X/month fixed charges.